isis/tests/PvlToJSONTests.cpp

```diff
@@ -0,0 +1,215 @@
+#include <nlohmann/json.hpp>
+
+#include "PvlToJSON.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "PvlObject.h"
+
+#include "gmock/gmock.h"
+
+using json = nlohmann::json;
+using namespace Isis;
+
+
+TEST(PvlToJSONTest, KeywordConversion) {
+  PvlKeyword testKey1("TestKey1", "A");
+  PvlKeyword testKey2("TestKey2", "1");
+  testKey2 += "2";
+
+  json testJson1 = pvlKeywordToJSON(testKey1);
+  json testJson2 = pvlKeywordToJSON(testKey2);
+
+  EXPECT_EQ(testJson1["Value"], testKey1[0].toStdString());
+  EXPECT_EQ(testJson2["Value"][0], testKey2[0].toStdString());
+  EXPECT_EQ(testJson2["Value"][1], testKey2[1].toStdString());
+}
+
+
+TEST(PvlToJSONTest, KeywordCommentConversion) {
+  PvlKeyword testKey1("TestKey1", "This keyword has 1 comment");
+  testKey1.addComment("Test comment");
+  PvlKeyword testKey2("TestKey2", "This keyword has multiple comments");
+  testKey2.addComment("First comment");
+  testKey2.addComment("Second comment");
+  PvlKeyword testKey3("TestKey3", "This keywords has no comments");
+
+  json testJson1 = pvlKeywordToJSON(testKey1);
+  json testJson2 = pvlKeywordToJSON(testKey2);
+  json testJson3 = pvlKeywordToJSON(testKey3);
+
+  EXPECT_EQ(testJson1["Comment"], testKey1.comment(0).toStdString());
```

Comment:
Should "testJson1["Comment"][0]" be tested also? That is, does [0] still work when there is only one element.

Reply:
[0] does not work when there is only 1 value.